app/models/user.py

```diff
@@ -352,3 +355,19 @@ def __setattr__(self, name, value):
 @event.listens_for(User, 'init')
 def receive_init(target, args, kwargs):
     target.signup_at = datetime.now(pytz.utc)
+
+
+@event.listens_for(User, 'after_update')
```

Comment:
get_history retutn this 

`History(added=[u'me@ipiyush.comss'], unchanged=(), deleted=[u'me@ipiyush.coms'])`